### DISCUSSION TITLE
Modify highlights for onedark AND onelight

### DISCUSSION THREAD

**Discussion by LamprosPitsillos** [olimorris/onedarkpro.nvim#158] [Q&A] [answered]
```lua
            highlights = {
                ["@variable"] = { fg = "${white}", style = "italic" }
            },
```
I tried playing with the settings to make variables not red ( imo in some languages makes almost everything red ie. lua )
Which somewhat worked but when i change to onelight , it becomes invisible cause of the white background...
How can i configure this to change for each variant ? 
thanks!

**Comment by olimorris** [comment, marked as answer]
This is covered in the [readme](https://github.com/olimorris/onedarkpro.nvim#configuring-highlight-groups). See the **Specifying highlight attributes by theme or background** section.
